examples/index.css

```diff
@@ -27,6 +27,13 @@ body {
     line-height: 1.4;
 }
 
+/*
+    Make trailing whitespaces visible since they can be quite confusing.
+*/
+[data-editable] {
+    white-space: pre-wrap;
+  }
```

Comment:
Space before ` }` and I’m very tempted to make this part of PR https://github.com/livingdocsIO/editable.js/pull/246 (which has already been merged 😳).